include/fiddle/postprocess/fiber_network.h

```diff
@@ -0,0 +1,64 @@
+#ifndef included_fiddle_postprocess_fiber_network_h
+#define included_fiddle_postprocess_fiber_network_h

```

Comment:
this should be in `mechanics`, not `postprocess` (we aren't postprocessing anything)